docs/tutorials/user-transpiled-circuits.ipynb

```diff
@@ -0,0 +1,2496 @@
+       "      <use xlink:href=\"#IBMPlexSans-72\" x=\"408.899902\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-69\" x=\"445.599899\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-62\" x=\"470.599884\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-75\" x=\"528.599869\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-74\" x=\"585.399857\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-69\" x=\"620.499847\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-6f\" x=\"645.499832\"/>\n",
+       "      <use xlink:href=\"#IBMPlexSans-6e\" x=\"701.499817\"/>\n",
+       "     </g>\n",
+       "    </g>\n",
+       "   </g>\n",
+       "  </g>\n",
+       " </g>\n",
+       " <defs>\n",
+       "  <clipPath id=\"p91ef219e8a\">\n",
+       "   <rect x=\"60.368437\" y=\"7.2\" width=\"502.2\" height=\"271.8\"/>\n",
+       "  </clipPath>\n",
+       " </defs>\n",
+       "</svg>\n"
+      ],
+      "text/plain": [
+       "<Figure size 648x360 with 1 Axes>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.visualization import plot_histogram\n",
+    "plot_histogram(result.quasi_dists+[ideal_distribution],\n",
+    "               sort='value_desc', bar_labels=False,\n",
+    "               legend=['optimization_level=0',\n",
+    "                       'optimization_level=3',\n",
+    "                       'ideal distribution'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'qiskit-terra': '0.19.2', 'qiskit-aer': '0.9.1', 'qiskit-ignis': '0.7.0', 'qiskit-ibmq-provider': '0.18.3', 'qiskit-aqua': '0.9.5', 'qiskit': '0.34.0', 'qiskit-nature': '0.3.0', 'qiskit-finance': '0.2.1', 'qiskit-optimization': '0.2.3', 'qiskit-machine-learning': '0.2.1'}"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import qiskit\n",
+    "qiskit.__qiskit_version__"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'0.2.0'"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import qiskit_ibm_runtime\n",
+    "qiskit_ibm_runtime.version.get_version_info()"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
Do we want to add another cell at the bottom with qiskit copyright notice like in other tutorials?
```
from qiskit.tools.jupyter import *

%qiskit_copyright
```

Reply:
Yes! Will add, thanks